GVRf/Framework/framework/src/main/java/org/gearvrf/GVRContext.java

```diff
@@ -524,9 +537,23 @@ public GVRSceneObject getAssimpModel(String assetRelativeFilename)
      */
     public GVRSceneObject getAssimpModel(String assetRelativeFilename,
             EnumSet<GVRImportSettings> settings) throws IOException {
-        return GVRImporter.getAssimpModel(this, assetRelativeFilename, settings);
+        return getAssimpModel(assetRelativeFilename, settings, null);
```

Comment:
I am removing this function. I noticed it has already been deprecated so we don’t need yet another version of it.

From: rahul [mailto:notifications@github.com]
Sent: Thursday, June 2, 2016 5:44 PM
To: Samsung/GearVRf <GearVRf@noreply.github.com>
Cc: Nola Donato <nola.donato@samsung.com>; Author <author@noreply.github.com>
Subject: Re: [Samsung/GearVRf] Wait for textures to be loaded before adding model to the scene (#597)


In GVRf/Framework/framework/src/main/java/org/gearvrf/GVRContext.java<https://github.com/Samsung/GearVRf/pull/597#discussion_r65642679>:

> @@ -524,9 +537,23 @@ public GVRSceneObject getAssimpModel(String assetRelativeFilename)

>       */

>      public GVRSceneObject getAssimpModel(String assetRelativeFilename,

>              EnumSet<GVRImportSettings> settings) throws IOException {

> -        return GVRImporter.getAssimpModel(this, assetRelativeFilename, settings);

> +        return getAssimpModel(assetRelativeFilename, settings, null);

passing a null argument to IAssetEvents would result in an NPE inside the function call.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Samsung/GearVRf/pull/597/files/fb58b0721681e130ce48eec2b79e1fc44b7ecf78#r65642679>, or mute the thread<https://github.com/notifications/unsubscribe/AQw1S9JUmuN4ZxYfIvy-YxMTit1GZFMEks5qH3jIgaJpZM4ItA3y>.